Tests/conf.json

```
@@ -3329,7 +3330,7 @@
         "Cymulate": "Partner didn't provided test playbook",
         "Lastline v2": "Temporary skipping, due to quota issues, in order to merge a PR",
         "Palo Alto Minemeld": "Issue #26878",
-        "AttackIQFireDrill": "License issues #29774"
+        "AttackIQFireDrill": "License issues #29774",
```

I think this will fail 
```suggestion
        "AttackIQFireDrill": "License issues #29774"
```